app/policies/exam_template_policy.rb

```diff
@@ -0,0 +1,16 @@
+# Policy for Exam templates controller.
+class ExamTemplatePolicy < ApplicationPolicy
+  default_rule :manage?
+  alias_rule :index?, :create?, :destroy?, :update?, :generate?,
+             :split?, :add_fields?, :view_logs?, :download?, :download_generate?,
+             :download_raw_split_file?, :show_cover?, to: :modify?
+
+  def manage?
+    user.admin?
+  end
+
+  # Only admin and authorized grader can manage or modify exam templates
+  def modify?
```

Comment:
You can remove this rule (and the `alias_rule` above), and have the default rule `manage` refer to the `manage_assessments?` permission.

(Like `AnnotationCategoryPolicy`.)